setup.py

```diff
@@ -23,10 +23,7 @@
     "nbsphinx>=0.3",
     "aioresponses~=0.6.0",
     "moto~=1.3.8",
-    "fakeredis~=1.0",
-    # upstream dep from fakeredis, should be removed if fakeredis properly depends on
-    # at least 1.12
-    "six>=1.12.0",
+    "fakeredis~=1.0.5",
```

Comment:
the requirement is a bit tight, should probably be smth like `fakeredis~=1.0,>=1.0.5`

Reply:
actually 1.0 already depends on `six>=1.12`, so we don't have to do anything